lachesis-rs/src/server/ws_message.rs

```diff
@@ -0,0 +1,19 @@
+use actix::*;
+
+use bytes::Bytes;
+
+use actix_web::Binary;
+use bincode::serialize;
+
+#[derive(Message, Serialize, Deserialize, Clone, Debug)]
+pub enum InternodeMessage {
+    SyncRequest,
+    SyncResponse,
+}
+
+impl Into<Binary> for InternodeMessage {
+    fn into(self) -> Binary {
+        let encoded: Vec<u8> = serialize(&self).unwrap();
```

Comment:
Looked into that but I can't use it because the library requires Into to be implemented (can't fail). I have changed it to log and error and return an empty vec.